src/state_transition/types/beacon_block.zig

```diff
@@ -185,81 +212,59 @@ pub const BeaconBlockBody = union(enum) {
     // bellatrix fields
     pub fn getExecutionPayload(self: *const BeaconBlockBody) ExecutionPayload {
         return switch (self.*) {
-            inline .phase0, .altair => @panic("ExecutionPayload is not available in phase0 or altair"),
             .bellatrix => |body| .{ .bellatrix = &body.execution_payload },
             .capella => |body| .{ .capella = &body.execution_payload },
             .deneb => |body| .{ .deneb = &body.execution_payload },
             .electra => |body| .{ .electra = &body.execution_payload },
+            .phase0, .altair => panic("ExecutionPayload is not available in {}", .{self}),
         };
     }
 
     // capella fields
     pub fn getBlsToExecutionChanges(self: *const BeaconBlockBody) []SignedBLSToExecutionChange {
         return switch (self.*) {
-            .phase0,
-            => @panic("BlsToExecutionChanges is not available in phase0"),
-            .altair => @panic("BlsToExecutionChanges is not available in altair"),
-            .bellatrix => @panic("BlsToExecutionChanges is not available in bellatrix"),
             .capella => |body| body.bls_to_execution_changes.items,
             .deneb => |body| body.bls_to_execution_changes.items,
             .electra => |body| body.bls_to_execution_changes.items,
+            .phase0, .altair, .bellatrix => panic("BlsToExecutionChanges is not available in {}", .{self}),
```

Comment:
it's a bit simpler to use `else`, same to below functions

Reply:
addressed in [d0876b5](https://github.com/ChainSafe/state-transition-z/pull/29/commits/d0876b59fca94734d211cf96224284e9d6c0548a)